hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionMetrics.java

```diff
@@ -1,13 +1,17 @@
 package pl.allegro.tech.hermes.api;
 
+import java.util.List;
 import java.util.Objects;
 
 public class SubscriptionMetrics {
     private long delivered;
     private long discarded;
     private long inflight;
-    private String rate = "0.0";
+    private String timeoutsM1;
```

Comment:
maybe just make it `timeouts` without `M1`, same as rate has no description, this is implementation detail and should be highlighted in documentation, not code